ion_schema_2_0/constraints/byte_length.isl

```diff
@@ -0,0 +1,103 @@
+$ion_schema_2_0
+
+type::{
+  name: 'byte_length: 5',
```

Comment:
(Suggestion) Should we just name all of these tests as snake case `byte_length_5`? It just confuses with the actual constraint when you look at the test.

Reply:
Sure, I can make that change.

Reply:
Created #34 to follow up on this once I've wrapped up all of the other test case PRs that I have in flight.